.golangci.yml

```diff
@@ -12,17 +12,17 @@ run:
 linters:
   disable-all: true
   enable:
+    - errcheck
     - gofmt
     - staticcheck
-    - deadcode
     - goconst
     - goimports
     - gosimple
     - govet
     - ineffassign
     - misspell
     - unconvert
-    - varcheck
+    - unused
```

Comment:
Fixing this warning:
```
WARN [runner] The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused. 
WARN [runner] The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused.
```